README.md

```diff
@@ -66,15 +66,25 @@ vars:
 ```
 
 ## (Optional) Step 5: Additional configurations
-<details><summary>Expand to view configurations</summary>
-    
+
 ### Add passthrough columns
 This package includes all source columns defined in the staging models. However, the `stg_zendesk__ticket` model allows for additional columns to be added using a pass-through column variable. This is extremely useful if you'd like to include custom fields to the package.
 ```yml
 vars:
   zendesk__ticket_passthrough_columns: [account_custom_field_1, account_custom_field_2]
 ```
 
+### Mark Former Internal Users as Agents
```

Comment:
reminder to myself to add this to the transform README (no release needed there tho) after approval